model/waterfall.go

```diff
@@ -522,3 +527,30 @@ func GetOlderActiveWaterfallVersion(ctx context.Context, projectId string, versi
 	}
 	return &res[0], nil
 }
+
+func GetVersionRevisionOrder(ctx context.Context, revision string, projectId string, limit int) (int, error) {
+	if len(revision) < minRevisionLength {
+		return 0, errors.New(fmt.Sprintf("at least %d characters must be provided for the revision", minRevisionLength))
+	}
+
+	found, err := VersionFindOne(ctx, VersionByProjectIdAndRevisionPrefix(projectId, revision).WithFields(VersionRevisionOrderNumberKey))
+	if err != nil {
+		return 0, errors.New(fmt.Sprintf("finding version with revision '%s': %s", revision, err.Error()))
+	} else if found == nil {
+		return 0, errors.New(fmt.Sprintf("version with revision '%s' not found", revision))
+	}
+	// Offset the order number so the specified revision lands nearer to the center of the page.
```

Comment:
Since this is UI specific, would it make sense to have this function return just the VersionRevisionOrder and do this calculation somewhere else? Alternatively, can we rename it to something that more explicitly shows that it won't simply get the version revision order?

Reply:
all the functions in this file are actually UI-specific so I think it's okay!

Reply:
That makes sense. In that case, I think explaining the offset more in the function name and comment would be helpful. 
thank you!

Reply:
ok i will add that!